src/tasks/deploy.mk

```
@@ -1,11 +1,19 @@
 #Must be above deplo%
 deploy-asset%: ## deploy-assets: uploads static files such as CSS and JS to S3 based on the contents of a manifest file
-	@if [ -e public/manifest.json ]; then\
-		nht deploy-hashed-assets --monitor-assets --manifest-file manifest.json --assets-are-hashed --destination-directory page-kit;\
+	@if [ -e public/manifest.json ]; then \
+		if [ -e .circleci/shared-helpers ]; then \
+			.circleci/shared-helpers/helper-install-awscli; \
```

```suggestion
			.circleci/shared-helpers/helper-install-awscli \
```

This isn't the end of the statement